tailwind.config.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
           600: '#783D86',
         },
         'du-gray': '#F7FAFC',
+        'indigo':{
```

Comment:
use the `du-` prefix for indigo so we don't override default tailwind styles if one exists